service/src/main/java/uk/gov/hmcts/reform/fpl/service/DocumentSenderService.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.SentDocument;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+
+import java.time.Clock;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.time.LocalDateTime.now;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class DocumentSenderService {
+
+    private final Clock clock;
+    private final DateFormatterService dateFormatterService;
+
+    public List<SentDocument> send(DocumentReference mainDocument,
+                                   Long ccdCaseNumber,
+                                   String familyManCaseNumber,
+                                   List<Representative> representativesServedByPost) {
+        //TODO  generate, stitch and send here
+
+        List<SentDocument> printedDocuments = new ArrayList<>();
```

Comment:
let's change variable to match type: sentDocuments

Reply:
done